### DISCUSSION TITLE
Configuración de certificado. Fondo e imagenes

### DISCUSSION THREAD

**Discussion by fabianinti** [chamilo/chamilo-lms#5250] [General]
Buenas a todos. Necesito ayuda con la configuración de mi certificado, por favor. Cuando configuro mi certificado en HTML y le pongo una imagen de fondo o utilizo el fondo predeterminado que nos otorga (fondo plomo en el DIV )y al descargar en PDFno me sale el fondo que se le puso anteriormente, también al visualizarlo a través del navegador no me sale las imágenes. Necesito ayuda, Por favor. Gracias.

![Captura de pantalla 2024-03-15 012059](https://github.com/chamilo/chamilo-lms/assets/163497982/0fe3cb42-8c41-429c-b55a-e82672e7472b)
![Captura de pantalla 2024-03-15 012129](https://github.com/chamilo/chamilo-lms/assets/163497982/e950ea31-a564-437f-b896-2690f8d0f015)

**Comment by contidos** [comment]
Hola Fabian, Normalmente cuando no salen las imágenes en el certificado es porque están subidas a un directorio al que ese curso o sesión no tiene acceso. Revisa esa ruta.

**Reply by fabianinti** [reply]
Muchas gracias, las rutas están correctas, si lo exportamos en pdf me sale normal las imágenes, pero si lo visualizamos desde el navegador no me sale ninguna de estas.
